src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -57,11 +57,13 @@ private int getTableRowsCount(String tableName) throws SQLException {
         }
     }
 
-    public void closeConnection() {
+    public void close() {
         try {
-            connection.close();
-        } catch (SQLException e) {
-            System.out.println(e.getMessage());
+            if(connection != null && !connection.isClosed()) {
+                connection.close();
+            }
+        } catch (SQLException | NullPointerException sqlException) {
```

Comment:
Why you decide to handle SQLException here? You'll catch it, write "The connection is not established!" to System.err and what than? Also, handling of NPE is a bad practice (http://bit.ly/2z3Cwjc), becides after 'if(connection != null && !connection.isClosed())' you won't have NPE in any case

Reply:
Of course you're right, I need to sleep (